osh/glob_.py

```diff
@@ -455,33 +501,37 @@ def _Glob(self, arg, out):
 
         return 0
 
-    def Expand(self, arg, out):
-        # type: (str, List[str]) -> int
+    def Expand(self, arg, out, blame_loc):
+        # type: (str, List[str], loc_t) -> int
         """Given a string that could be a glob, append a list of strings to
-        'out'.
+        'out' when glob expansion happens.
 
         Returns:
-          Number of items appended, or -1 for fatal failglob error.
+          Number of items appended, or -1 when glob expansion did not happen.
```

Comment:
It should also be noted that I changed the meaning of `return -1` of this function (`globber.Expand`).